gulp/tasks/default.js

```diff
@@ -4,6 +4,7 @@ const runSequence = require('run-sequence');
 gulp.task('default', () => {
   return runSequence.use(gulp)(
     'clean',
-    'build'
+    'build',
+    'build_es6'
```

Comment:
I know it's kind of too late now, but I think "modern" would have been a better suffix that would stand time as I can see us moving what we transpile.

Reply:
Maybe change to es5 and un-suffixed?

Reply:
Oh I like that more. ES5 is a fixed goal.